src/shipit_code_coverage_crawler/tests/test_example.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+
+
+def test_example(app):
+    assert True
```

Comment:
I also need to remove this file, yes?

Reply:
This is a placeholder test, but we can keep it and then modify it when we'll actually write tests.